.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        BUILD_TYPE: [Release, Debug]
+      
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    # install dependencies
+    - name: dependencies
+      run: sudo apt-get update && sudo apt-get install -yq openmpi-bin libfftw3-dev python3-dev libboost-all-dev python3-h5py
+
+    - name: Create Build Environment
+      # Some projects don't allow in-source building, so create a separate build directory
+      # We'll use this as our working directory for all subsequent commands
+      run: cmake -E make_directory ${{github.workspace}}/build
```

Comment:
You could drop that step if you run `cmake -B ..` below.

Reply:
Yes, still old template. I will rework it.